docs/docsite/rst/dev_guide/style_guide/index.rst

```diff
@@ -171,13 +171,33 @@ The second example adds custom text for the link.
 Adding links to modules and plugins
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-* Module links use the module name followed by ``_module`` for the anchor.
-* Plugin links use the plugin name followed by the plugin type. For example, :ref:`enable become plugin <enable_become>`).
+Ansible 2.10 and later require the extended Fully Qualified Collection Name (FQCN) as part of the links:
+
+.. code-block:: text
+
+  ansible_collections. + FQCN + _module
+
+For example:
+
+  .. code-block:: rst
+
+   :ref:`ansible.builtin.enable become plugin <ansible_collections.ansible.builtin.enable_become>`
```

Comment:
The enable become plugin is part of ansible.netcommon, not ansible.builtin. Since we want an example of an ansible.builtin plugin, how about:
```suggestion
   :ref:`ansible.builtin.first_found lookup plugin <ansible_collections.ansible.builtin.first_found_lookup>`
```